setup.py

```diff
@@ -6,21 +6,42 @@
 GNU AFFERO GENERAL PUBLIC LICENSE Version 3.0 (AGPL).
 See LICENSE.txt for details.
 """
+import pathlib
```

Comment:
We should probably use `os.path` here to be backwards compatible with python2, since this will be used in python2 application in the foreseeable future. See also https://github.com/openstax/cnx-easybake/blob/f8edf018fb7499f6f18af0145c326b93a737a782/setup.py#L15-L21

Reply:
Fixed.